### DISCUSSION TITLE
GitPod development environment

### DISCUSSION THREAD

**Discussion by TiagoCavalcante** [Mathics3/mathics-core#113] [Ideas]
[GitPod](https://www.gitpod.io/) is used in Symja, it allows easily start contributing to any project.

We could add a .gitpod.yml file and "Open in GitPod" button.
GitPod allows us to use a custom Docker image and initialization commands, so we could create a .gitpod.yml easily.

**Comment by TiagoCavalcante** [comment]
@rocky @mmatera what do you think?

**Reply by rocky** [reply]
I don't have an opinion, but if you want to try it, go ahead and let's see how it goes.

**Comment by TiagoCavalcante** [comment]
Linked PR: #114

**Reply by rocky** [reply]
Does gitpod work on the branch?

**Reply by TiagoCavalcante** [reply]
It works in my fork, and that is up-to-date with this repo, so merging is not necessary for testing. (there's a link in the PR for testing, with GitPod, a git pull is not even necessary)
